---
title: IBMQAccountMultipleCredentialsFound
description: API reference for qiskit.providers.ibmq.IBMQAccountMultipleCredentialsFound
in_page_toc_min_heading_level: 1
python_api_type: exception
python_api_name: qiskit.providers.ibmq.IBMQAccountMultipleCredentialsFound
---

# IBMQAccountMultipleCredentialsFound

<span id="qiskit.providers.ibmq.IBMQAccountMultipleCredentialsFound" />

`IBMQAccountMultipleCredentialsFound(*message)`

Errors raised when multiple credentials are found.

Set the error message.

### with\_traceback

<span id="qiskit.providers.ibmq.IBMQAccountMultipleCredentialsFound.with_traceback" />

`with_traceback()`

Exception.with\_traceback(tb) â€“ set self.\_\_traceback\_\_ to tb and return self.

